src/app/shared/modal/modal-login/modal-login.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+
+import { AuthenticationService } from '../../services/authentication.service';
+import { User } from '../../interfaces/user.interface.';
+
+@Component({
+  selector: 'app-modal-login',
+  templateUrl: './modal-login.component.html',
+  styleUrls: ['./modal-login.component.css']
+})
+export class ModalLoginComponent implements OnInit {
+  @Input() appHideIfAuthenticated!: boolean;
+  @Output() userLogged: EventEmitter<void> = new EventEmitter<void>();
```

Comment:
Seems like you don`t need the **appHideIfAuthenticated** Input
And try to create the LoginComponent and keep the logic of logging there

Reply:
Yes, I don't need it, I just leaved it unintentionally after my some dumb exercises)